2022/09.md

```diff
@@ -95,7 +95,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 2     | 60m     | [Array.fromAsync](https://github.com/tc39/proposal-array-from-async) for Stage 3 (slides TBD) | J. S. Choi |
     |   | 2     | 60m     | Set methods discussion ([slides](https://docs.google.com/presentation/d/1HCqPMsWiTtsn92gA3b1luVpnVHWVVR0iKaAE0marxkA)) | Kevin Gibbons |
     |   | 2     | 60m     | [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management) for Stage 3 ([slides](https://1drv.ms/p/s!AjgWTO11Fk-TkoEvgEld8y2bGCjTlw?e=ySkk5z), [spec](https://tc39.es/proposal-explicit-resource-management/)) | Ron Buckton |
-    |   | 1     | 15m     | [Iteration helpers as functions](https://github.com/rauschma/iterable) for stage 1 (slides TBD) | Axel Rauschmayer |
+    |   | –     | 15m     | Discussion: switching [iterator helpers](https://github.com/rauschma/iterable) to functions ([slides](https://speakerdeck.com/rauschma/iteration-helper-functions)) | Axel Rauschmayer |
```

Comment:
A 15 minute timebox is *very* short for this group. New attendees often drastically underestimate their timeboxes. We are pretty strict about keeping topics within the timebox, so I would recommend significantly expanding it, probably to something like 60m. Up to you though.

Reply:
Good point. I changed it to 45min (15min presentation, 30min discussion).